source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/__tests__/archive-acceptance-test.tsx

```diff
@@ -24,6 +24,7 @@ describe('Acceptance | Archive Project', () => {
 
   describe('an active project exists', () => {
     beforeEach(function() {
+      this.mockGet(200, 'users', { data: [] });
```

Comment:
should not start with /users?

Reply:
turns out it's not required, because mockGet defines a namespace of `api` rather than doing string interpolation